examples/mario/play_mario.html

```diff
@@ -4,15 +4,15 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width,initial-scale=1" />
 
-    <title>Svelte app</title>
+    <title>Mrio Game - pyscript</title>
 
     <link rel="icon" type="image/png" href="../favicon.png" />
-    <link rel="stylesheet" href="../https://pyscript.net/alpha/pyscript.css" />
+    <link rel="stylesheet" href="https://pyscript.net/alpha/pyscript.css" />
 
-    <script defer src="../https://pyscript.net/alpha/pyscript.js"></script>
+    <script defer src="https://pyscript.net/alpha/pyscript.js"></script>
   </head>
 
-  <body>
+  <body allow="autoplay; fullscreen">
```

Comment:
why this change?

Reply:
here I allowed autoplay for the people who accept this option from the browser settings